catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/PipelineRepository.java

```diff
@@ -359,7 +359,9 @@ private void updateTaskDescription(Task origTask, Task updatedTask) throws JsonP
         return;
       }
       // Don't record a change if descriptions are the same
-      if (!origTask.getDescription().equals(updatedTask.getDescription())) {
+      if (origTask != null
+          && origTask.getDescription() != null
```

Comment:
it should be "Or" condition here, if the task doesn't have any description but the updatedTask has description we should set that

Reply:
Lets also add a unit test for this